llvm/lib/Target/SystemZ/SystemZISelLowering.cpp

```diff
@@ -2918,16 +3049,17 @@ static Comparison getCmp(SelectionDAG &DAG, SDValue CmpOp0, SDValue CmpOp1,
                          bool IsSignaling = false) {
   if (CmpOp1.getOpcode() == ISD::Constant) {
     assert(!Chain);
-    uint64_t Constant = cast<ConstantSDNode>(CmpOp1)->getZExtValue();
     unsigned Opcode, CCValid;
```

Comment:
Was 'Constant = ...' removed on purpose? (I would think that normally it would be factored out like it was, but maybe doesn't matter)

Reply:
`getZExtValue` will crash when the constant is 128-bit, which can now happen.  Therefore I'm now only evaluating it in code paths where we know the constant cannot be 128-bit.